nbnovnc/handlers.py

```diff
@@ -34,9 +34,9 @@ def get_cmd(self):
 class NBNoVNC(Configurable):
     geometry = Unicode(u"1024x768", help="Desktop geometry.", config=True)
     depth = Integer(24, help="Desktop display depth.", config=True)
-    novnc_directory = Unicode(u"/usr/share/novnc", config=True,
+    novnc_directory = Unicode(u"/usr/share/novnc/", config=True,
         help="Path to noVNC web assets.")
-    vnc_command = Unicode(u"xinit -- /usr/bin/Xtightvnc :{display} -geometry {geometry} -depth {depth}", config=True,
+    vnc_command = Unicode(u"xinit -- /usr/bin/Xtigervnc :99 -geometry {geometry} -depth {depth}  -SecurityTypes None", config=True,
```

Comment:
@djangoliv Looks like the display is still hard coded. Everything else looks great!

Reply:
updated. Thanks